examples/testing-module/Makefile

```diff
@@ -76,6 +76,10 @@ check-memcached-both: prepare-docker prepare-nspawn
 	cd ../memcached; MODULE=docker $(CMD) sanity1.py
 	cd ../memcached; MODULE=nspawn $(CMD) sanity1.py
 
+check-openshift-memcached:
+	sudo oc cluster up
```

Comment:
this is not part of ``mtf-env-set`` tool? for **openshift** support (same for down command)

Reply:
By the way this is kind of problematic. `mtf-env-set` installs some packages from Fedora repository. And OpenShift is not available in Ubuntu.

Reply:
yep, make sense